src/functions/assertions/FileContentMatchMultilineExactly.ps1

```diff
@@ -60,9 +60,16 @@
         }
     }
 
-    return [PSCustomObject] @{
+    $ExpectedValue = $ExpectedContent
+
+    return [Pester.ShouldResult] @{
         Succeeded      = $succeeded
         FailureMessage = $failureMessage
+        ExpectResult   = @{
```

Comment:
Here it populates the values no matter if the should fails, in some other places it only populates them when should fails. What is the desired way?

Reply:
Defer to @fflaten but seems to me it makes sense to not populate it if it's successful for minor performance benefit, the only use that info would be is troubleshooting (e.g. it said successful but it shouldn't have been)

Reply:
All assertions should now be adjusted to not include this property on a success. Maybe we need tests for this?